pom.xml

```diff
@@ -7,398 +7,24 @@
-      </modules>
-    </profile>
-    <profile>
-      <id>batch</id>
-      <modules>
-        <module>sdk/batch</module>
-      </modules>
-    </profile>
-    <profile>
-      <id>cognitiveservices</id>
-      <modules>
-        <module>sdk/cognitiveservices</module>
-      </modules>
-    </profile>
-    <profile>
-      <id>core</id>
-      <modules>
-        <module>sdk/core</module>
-      </modules>
-    </profile>
-    <profile>
-      <id>cosmos</id>
-      <modules>
-        <module>sdk/cosmos</module>
-      </modules>
-    </profile>
-    <profile>
-      <id>eventgrid</id>
-      <modules>
-        <module>sdk/eventgrid</module>
-      </modules>
-    </profile>
-    <profile>
-      <id>eventhubs</id>
-      <modules>
-        <module>sdk/eventhubs</module>
-      </modules>
-    </profile>
-    <profile>
-      <id>identity</id>
-      <modules>
-        <module>sdk/identity</module>
-      </modules>
-    </profile>
-    <profile>
-      <id>keyvault</id>
-      <modules>
-        <module>sdk/keyvault</module>
-      </modules>
-    </profile>
-    <profile>
-      <id>loganalytics</id>
-      <modules>
-        <module>sdk/loganalytics</module>
-      </modules>
-    </profile>
-    <profile>
-      <id>mediaservices</id>
-      <modules>
-        <module>sdk/mediaservices</module>
-      </modules>
-    </profile>
-    <profile>
-      <id>parents</id>
-      <modules>
-        <module>sdk/parents</module>
-      </modules>
-    </profile>
-    <profile>
-      <id>search</id>
-      <modules>
-        <module>sdk/search</module>
-      </modules>
-    </profile>
-    <profile>
-      <id>servicebus</id>
-      <modules>
-        <module>sdk/servicebus</module>
-      </modules>
-    </profile>
-    <profile>
-      <id>storage</id>
-      <modules>
-        <module>sdk/storage</module>
-      </modules>
-    </profile>
-    <profile>
-      <id>template</id>
-      <modules>
-        <module>sdk/template</module>
-      </modules>
-    </profile>
-    <profile>
-      <id>all</id>
-       <activation>
-         <activeByDefault>true</activeByDefault>
-       </activation>
-      <modules>
-        <module>sdk/appconfiguration</module>
-        <module>sdk/applicationinsights</module>
-        <module>sdk/authorization</module>
-        <module>sdk/batch</module>
-        <module>sdk/cognitiveservices</module>
-        <module>sdk/core</module>
-        <module>sdk/cosmos</module>
-        <module>sdk/eventgrid</module>
-        <module>sdk/eventhubs</module>
-        <module>sdk/identity</module>
-        <module>sdk/keyvault</module>
-        <module>sdk/loganalytics</module>
-        <module>sdk/mediaservices</module>
-        <module>sdk/parents</module>
-        <module>sdk/servicebus</module>
-        <module>sdk/storage</module>
-        <module>sdk/template</module>
-        <!--
-        <module>sdk/search</module>
-        -->
-      </modules>
-    </profile>
-
-  </profiles>
+  <modules>
```

Comment:
@mitchdenny isn't this going to cause us to have to publish this pom file?

Reply:
I've removed this file, but no anyway since its just an aggregator.

Reply:
Just a note, these profiles pom seems corresponding to this in spec https://github.com/Azure/azure-rest-api-specs/tree/master/profile

Reply:
I'm not familiar with how these spec files are processed does me moving the profiles directories cause a problem with code generation?

Reply:
Personally I think moving pom around would not affect published artifacts, so there should be no need to re-publish anything.

However there could be problem on `./pom.management.xml`, on swagger automation.

Current released autorest.java would generate service code with pom.xml refering to `<relativePath>../../../pom.management.xml</relativePath>`. And swagger automation uses that autorest.java.
If pom.management.xml moves without sync update/release on autorest.java, it could break swagger automation (though I think that automation on Java SDK is not a mandatory one for merge).

I am not 100% sure about this. Maybe you can just be prepared to at least temporary put back `./pom.management.xml` if we see swagger automation issue after this get merged.

Reply:
Perhaps we can just tweak the automation and merge the changes simultaneously. Where is the file that controls this relative path? (I'm not really across our code-gen at the moment).

Reply:
It is here https://github.com/Azure/autorest.java/blob/master/src/azurefluent/Model/CodeModelJvaf.cs#L145

Synced change might be hard. autorest.java need to get published to npm (not just code merged). Jianghao should know the details.